src/css.js

```diff
@@ -141,10 +147,14 @@ function getWidthOrHeight( elem, dimension, extra ) {
 	// Also use offsetWidth/offsetHeight for when box sizing is unreliable
 	// We use getClientRects() to check for hidden/disconnected.
 	// In those cases, the computed value can be trusted to be border-box
-	if ( ( isBorderBox && !support.boxSizingReliable() || val === "auto" ||
+	if ( ( !support.boxSizingReliable() &&
+		jQuery.css( elem, "boxSizing", false, styles ) === "border-box" ||
```

Comment:
You may get a size reduction by turning this bare condition into an assignment, matching its other two appearances.
```suggestion
		(isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box") ||
```

Reply:
@gibson042 This `if` is an alternative of conditions and this is just the first of them. Doing the assignment there would mean the value is not set properly for non-IE browsers that fulfill one of the other checks.

Reply:
Right, and that's fine. This is just about compressibility.

Reply:
Ah, I see, you meant adding the assignment without actually using it later. Let me check.

Reply:
@gibson042 That's still `+16` bytes, exactly as my first version of the PR. With @timmywil's suggestion of just using `isBorderBox` here we get down to `+15` bytes. That reminds the best option then.